telegram/_files/file.py

```diff
@@ -35,18 +35,22 @@
 
 class File(TelegramObject):
     """
-    This object represents a file ready to be downloaded. The file can be downloaded with
-    :attr:`download`. It is guaranteed that the link will be valid for at least 1 hour. When the
-    link expires, a new one can be requested by calling :meth:`telegram.Bot.get_file`.
+    This object represents a file ready to be downloaded. The file can be e.g. downloaded with
+    :attr:`download_to_drive`. It is guaranteed that the link will be valid for at least 1 hour.
+    When the link expires, a new one can be requested by calling :meth:`telegram.Bot.get_file`.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
     considered equal, if their :attr:`file_unique_id` is equal.
 
+    .. versionchanged:: 20.0:
+        ``download`` was split into :meth:`download_to_drive` and :meth:`download_to_object`.
```

Comment:
```suggestion
        ``download`` was split into :meth:`download_to_drive` and :meth:`download_to_memory`.
```

Just pitching my suggestions once more :D For my feeling, "memory" gives you a better idea of what this is doing. with "object", I have to think about what this naming is trying to tell me …

Reply:
uh pfff okay.